server/controllers/entities/lib/create_wd_entity.ts

```diff
@@ -16,7 +16,7 @@ import type { SimplifiedQualifiers, WikimediaLanguageCode } from 'wikibase-sdk'
 
 const wmLanguageCodeByWdId = requireJson('wikidata-lang/mappings/wm_code_by_wd_id.json')
 
-const allowlistedEntityTypes = [ 'work', 'serie', 'human', 'publisher', 'collection', 'edition' ]
+export const allowlistedEntityTypes = [ 'work', 'serie', 'human', 'publisher', 'collection', 'edition' ]
```

Comment:
Why not reusing `allLocallyEditedEntitiesTypes`? It does feel okay to infer "editable entities types" === "creatable entities type". Too much DRYing? I can see our future self reusing `allowlistedEntityTypes` within other scopes than this current 'entity creation' scope, without paying attention

Reply:
I was making for `create_wd_entity.ts` to have its own list when that was only a subset of the possible local types, but now that's obsolete, so it can be fully removed: 32c74e220